### PR TITLE
Adjusted round 2 of various UI and UX tinkering

### PR DESCRIPTION
**Pull request by victorskl**
* Improved mainly focus around Run view
* Re-introduced Chip filter buttons for GDS and S3 DataTable
* Hide few routes, views and menu navigation that are needing
  more time to rethink better UI/UX flow
* Switched to Bootstrap theme

### CODE REVIEW

**Review by williamputraintan — APPROVED** [review]
I think the CSS might still need more tuning, but feature wise LGTM!

File: src/containers/gds/GDSDataTable/index.tsx [hunk 1]
```diff
@@ -60,24 +71,37 @@ function GDSDataTable({ defaultQueryParam }: Props) {
...
   return (
     <>
-      <DataSearchFilterButton
-        currentFilter={searchField}
-        handleFilterChange={(s: string) => setSearchField(s)}
-      />
-      <div className={isFetching || isLoading ? '' : 'hidden'}>
-        <CircularLoaderWithText text='Fetching data, please wait...' />
-      </div>
-      <div className={isFetching || isLoading ? 'hidden' : ''}>
```
[the remark this answers is not in the document]
Comment (victorskl) [reply src/containers/gds/GDSDataTable/index.tsx:72]: Huh! Right. I was puzzled why blocking with CircularLoader... that answered it. 
Let me double check on my local; to see whether I can reproduce.

[the remark this answers is not in the document]
Comment (williamputraintan) [reply src/containers/gds/GDSDataTable/index.tsx:72]: I deployed locally, seems to be working fine to me!

**Comment by victorskl** [comment]
Merging...

**Pull request merged**